unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/Tests/SettingsControlsShould.cs

```diff
@@ -263,22 +263,6 @@ public void ChangeMouseSensivityCorrectly()
                 "freeLookCamera.m_YAxis.m_MaxSpeed value mismatch");
         }
 
-        [Test]
-        public void ChangeMuteSoundCorrectly()
-        {
-            // Arrange
-            settingController = ScriptableObject.CreateInstance<MuteSoundControlController>();
-            settingController.Initialize();
-
-            // Act
-            bool newValue = true;
-            settingController.UpdateSetting(newValue);
-
-            // Assert
-            Assert.AreEqual(newValue, settingController.GetStoredValue(), "muteSound stored value mismatch");
-            Assert.AreEqual(newValue ? 1f : 0f, AudioListener.volume, "sfxVolume value mismatch");
-        }
-
```

Comment:
Could you create the corresponding test for each new audio setting control you have added? (`ChangeAvatarSFXVolumeCorrectly`, `ChangeChatSFXToggleCorrectly`, `ChangeMasterVolumeCorrectly`, `ChangeMusicVolumeCorrectly`, `ChangeSceneSFXVolumeCorrectly`, `ChangeUISFXVolumeCorrectly`).

Reply:
Added tests for all the controls. Currently, they only test their internal SettingsController value. Should they also be testing the final volume output? For example, some of them change the volume of AudioMixer groups - so should we test the value against these?

Reply:
It's ok just like you did it, thanks! :)